test/test_utils/sagemaker.py

```diff
@@ -314,6 +314,7 @@ def execute_local_tests(image, pytest_cache_params):
         ec2_conn.run(f"tar -xzf {sm_tests_tar_name}")
         kill_background_processes_and_run_apt_get_update(ec2_conn)
         with ec2_conn.cd(path):
+            print(f"Changed path to: {path}")
```

Comment:
Is this a temporary addition, or is it meant to be permanent? If permanent, please use `LOGGER` instead of `print` statements.

Reply:
Temporal. I want to see the directory switched to.